documentation/experimental_features.md

```diff
@@ -216,6 +216,18 @@ ResourceInstance.new(
 )
 ```
 
+### Accessing `ResourceInstance` object on a Target
+
+You can get a specific `ResourceInstance` object stored on a Target in a plan using the `resource()`
```

Comment:
```suggestion
You can retrieve a specific `ResourceInstance` object stored on a Target in a plan using the `resource()`
```
Get is a little ambiguous here. Does "retrieve" make sense?

Reply:
Sure, updated